lib/features/add_property/view/add_property_view.dart

```diff
@@ -1,14 +1,154 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
```

Comment:
Oops. the scope of this HS-133 focus on warning_lost data. you don't have to add UI for add property view.

Reply:
revert this class